inst/application/ui.R

```diff
@@ -278,21 +264,21 @@ fluidPage(
                              step = 500,
                              min = 0
                 ) %>%
-                  bs_embed_tooltip(title = IB$P3purchase, placement = "right"),
+                  bs_embed_tooltip_body(title = IB$P3purchase, placement = "right"),
                 numericInput("returnP3",
                              label = "Expected Return %",
                              value = value$p3_return,
                              step = 0.1,
                              min = 0
                 ) %>%
-                  bs_embed_tooltip(title = IB$returnP3, placement = "right")
+                  bs_embed_tooltip_body(title = IB$returnP3, placement = "right")
               )
             )
-          ) # end of 3rd Pillar boxPlus
-        ), # end of 3rd Pillar fluidRow
+          ), # end Pillar III SmaRPanel
```

Comment:
We should align this comment to the new pillar naming, and also align the `id` of the panels above.

Reply:
@RolandASc, I just realized I never pushed some updates I committed locally after merging the latest updates from develop back in May! I am fixing the mess!

Reply:
Fixed un-pushed conflicts in  8965b42

Reply:
Fixed panel ids in 
7f12f6f